README.md

```diff
@@ -28,8 +28,12 @@ Here’s a basic example of usage of the new API:
 ```javascript
 const wss = new WebSocketStream(url);
 const { readable } = await wss.connection;
-for await (const chunk of readable) {
-  await process(chunk);
+const reader = readable.getReader();
+while (true) {
+  const { value, done } = reader.read();
```

Comment:
```suggestion
  const { value, done } = await reader.read();
```

Reply:
@ricea I've tried to implement a [demo](https://glitch.com/~websocketstream-demo) with this new API. The `read()` method lacks an `await`.